skipruntime-ts/adapters/postgres/src/index.ts

```diff
@@ -144,28 +153,38 @@ FOR EACH ROW EXECUTE FUNCTION %I();`,
           ),
         );
       }
+    };
+
+    const listenToPgNotify = () => {
       this.client.on("notification", (msg) => {
         if (msg.channel == instance && msg.payload !== undefined) {
           const query = key.select(table, msg.payload);
           this.client.query(query).then(
             (changes) => {
               const k = key.type == "TEXT" ? msg.payload! : Number(msg.payload);
-              callbacks.update([[k, changes.rows as Json[]]], false);
-            },
-            (e: unknown) => {
-              console.error(`Error executing Postgres query: ${query}`);
-              throw e;
+              try {
+                callbacks.update([[k, changes.rows as Json[]]], false);
+              } catch (e: unknown) {
```

Comment:
Do you see this actually catching exceptions thrown from mapEntry? I have similar try-catch in uncommitted debugging exploration and do not see it actually catching them.

Reply:
I did in an earlier iteration, but it's been through a couple of those and may no longer be the case... testing/tweaking now to be sure.